config/datatypes_conf.xml.sample

```diff
@@ -443,8 +443,10 @@
     <sniffer type="galaxy.datatypes.tabular:Vcf"/>
     <sniffer type="galaxy.datatypes.binary:TwoBit"/>
     <sniffer type="galaxy.datatypes.binary:GeminiSQLite"/>
-    <sniffer type="galaxy.datatypes.proteomics:MzSQlite"/>
+    <sniffer type="galaxy.datatypes.binary:MzSQlite"/>
```

Comment:
When updating to the next Galaxy release, if there is a `config/datatypes_conf.xml` file (not the sample), these changes will have to be manually merged by the admin. It's not a showstopper anyway.